src/init.cpp

```diff
@@ -1276,6 +1276,69 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
                                      chainman, *node.mempool, ignores_incoming_txs);
     RegisterValidationInterface(node.peerman.get());
 
+    // Check port numbers
+    const bilingual_str port_error = _("Invalid port specified in ");
```

Comment:
```suggestion
    const bilingual_str port_error = _("Invalid port specified in %s: %s");
```

Other languages may need to change the ordering of parameters.

Reply:
Yeah, I did that initially but the linter complains, any tips?

Reply:
Check out the functions at the bottom of `src/util/error.cpp`

Reply:
Thanks for the review Luke, much appreciated